lib/putasset.js

```diff
@@ -17,6 +17,30 @@ module.exports = async (token, {owner, repo, tag, filename}) => {
         type,
         token,
     });
+
+    const file_basename= path.basename(filename);
+
+    const asset_id = await github.repos.listReleases(
+        { owner, repo, "per_page": 100, "page": 1 }
+    ).then(({ data }) =>{
+
+        const asset= data
+            .find(({ tag_name }) => tag_name === tag)
+            .assets
+            .find(({ name })=> name === file_basename)
+            ;
+
+        return !!asset?asset.id:undefined;
+
+    });
+
+    if( asset_id !== undefined ){
+
+        await github.repos.deleteReleaseAsset(
+            {owner, repo, asset_id}
+        );
+
+    }
```

Comment:
Could you please keep original code style

Reply:
ok